gradle.properties

```diff
@@ -1 +1 @@
-version = 1.5.0
+version = 1.6.0-SNAPSHOT
```

Comment:
Maybe the release task should bump version and add "-SNAPSHOT" as soon as the release is done (as would Maven do).